### ISSUE TITLE
XQuery Tutorial and Hmwk 1, and Project Midterms

### ISSUE THREAD

**Issue by ebeshero** [ebeshero/DHClass-Hub#161]
@ghbondar @brookestewart @nlottig94 @spadafour @RJP43 @mmm202 @laurenmcguigan @mjb232 @jlm323  Greetings from the Sigma Tau Delta Conference in Minneapolis! I've completed revisions to [our XQuery tutorial](http://newtfire.org/dh/explainXQuery.html) and to your homework assignment, [XQuery Ex 1](http://newtfire.org/dh/XQueryExercise1.html).

## Project Midterms
We ask that the project teams make substantial progress on your projects over the break, and to complete the midterm, we'll ask you to describe what you have accomplished and represented in your project website in some way. Your progress must involve:
1) markup of your texts 
2) schema development (Schematron)
3) some efforts (even preliminary) with extracting information from your markup to explore your research question. 
4) an explanation of what you plan to accomplish next.

**Comment by nlottig94** [comment]
For the Dickinson Project, @brookestewart and I are going to apply the Schematron to the poems and move some things around so there aren't so many random files in places. We are going to then try out some liminal to our code to see what kind of visualizations we can make with it. We haven't done much over break because we have both been away and unable to work on our computers, but out work will get finished!  Oh! And we were accepted to the Keystone DH Conference as a Long Paper!

**Comment by ebeshero** [comment]
@nlottig94 @brookestewart @RJP43 @ghbondar @spadafour @jlm323 @laurenmcguigan @mmm202 @mjb232 We are extending the due date of the project midterms until Monday, so that you will have time to complete some first large tasks that you're beginning. We instructors thought that seemed reasonable given the solid beginnings we're seeing right now, so we'd like you to keep working on this through the weekend. The midterms will be due with SVG Exercise 1.

**Comment by jlm323** [comment]
For the Decameron project, @laurenmcguigan @mmm202 and myself worked on marking up the text. We tagged the persNames in the floatingText, we tagged the persNames on the 10 storytellers, and we tagged the placeNames for the entire text respectively. Lauren also started developing our Schematron and has been looking into TEI element names for us to use. @mjb232 has worked on the prosopography and had begun creating our personography. We plan on continuing to markup the text's nested narrative, other persNames, and finding some interesting research questions to explore as we continue to read.
